### PR TITLE
fix build on windows 

### PR DESCRIPTION
**Pull request by grishasobol**
fix build on windows
fix problems with std feature and substrate + gear

### CODE REVIEW

File: lazy-pages/src/lib.rs [hunk 1]
```diff
@@ -56,6 +56,10 @@ thread_local! {
 /// instruction, which cause signal. Now memory which this instruction accesses
 /// is not protected and with correct data.
 extern "C" fn handle_sigsegv(_x: i32, info: *mut siginfo_t, _z: *mut c_void) {
+    if !cfg!(unix) {
+        panic!("unsupported");
```
Comment (breathx) [inline lazy-pages/src/lib.rs:60]: Isn't it too hard to panic in such case? What about placing here `return` statement?

Comment (NikVolf) [reply lazy-pages/src/lib.rs:60]: This code shouldn't ever hit @grishasobol ?

File: runtime-interface/src/lib.rs [hunk 2]
```diff
@@ -27,6 +27,48 @@ use gear_core::memory::PAGE_SIZE;
...
+
+#[cfg(not(unix))]
+unsafe fn sys_mprotect_wasm_pages(
+    from_ptr: u64,
+    pages_nums: &[u32],
+    prot_read: bool,
+    prot_write: bool,
+    prot_exec: bool,
+) {
+    panic!("unsupported");
```
Comment (breathx) [inline runtime-interface/src/lib.rs:69]: Still same question about panics

Comment (shamilsan) [reply runtime-interface/src/lib.rs:69]: Yes, if it is unreachable, better would be to use `unreachable!(message)`.

File: .github/workflows/win.yml [hunk 3]
```diff
@@ -36,3 +36,4 @@ jobs:
         with:
           command: build
           args: -p gear-node --release --target x86_64-pc-windows-msvc
+
```
Comment (shamilsan) [inline .github/workflows/win.yml:39]: Extra empty line?
```suggestion
```

Comment (grishasobol) [reply .github/workflows/win.yml]: Ok

File: runtime-interface/src/lib.rs (hunk 2 above)
Comment (grishasobol) [reply runtime-interface/src/lib.rs]: Ok

File: runtime-interface/src/lib.rs [hunk 4]
```diff
@@ -27,6 +27,48 @@ use gear_core::memory::PAGE_SIZE;
...
+
+#[cfg(not(unix))]
+unsafe fn sys_mprotect_wasm_pages(
+    from_ptr: u64,
+    pages_nums: &[u32],
+    prot_read: bool,
+    prot_write: bool,
+    prot_exec: bool,
+) {
+    unreachable!("unsupported OS for pages protectections");
```
Comment (shamilsan) [inline runtime-interface/src/lib.rs:69]: Typo?
```suggestion
    unreachable!("unsupported OS for pages protection");
```

**Pull request merged**